base/BUILD

```diff
@@ -36,3 +36,28 @@ structure_test(
     config = "testdata/base.yaml",
     image = ":base",
 )
+
+# For convenience, rename busybox-x86_64 to busybox.
+genrule(
+    name = "_busybox",
+    srcs = ["@busybox//file"],
+    outs = ["busybox"],
+    cmd = "cp $(<) $(@)",
+    visibility = ["//visibility:private"],
+)
+
+# A debug image with busybox available.
+docker_build(
```

Comment:
You should also include symlinks that alias `busybox` as the various commands it implements.

Perhaps to make it easier to include in different contexts we should define a `pkg_tar` rule for the files and symlinks and then it can be included by users in an adhoc way?

Do you want to start adding ":debug" targets to each of the base runtimes that includes this and publishing them?

Looking at what Alpine aliases to `busybox` in its base layer:

```
bin/ash
bin/base64
bin/bbconfig
bin/busybox
bin/cat
bin/catv
bin/chgrp
bin/chmod
bin/chown
bin/conspy
bin/cp
bin/cpio
bin/date
bin/dd
bin/df
bin/dmesg
bin/dnsdomainname
bin/dumpkmap
bin/echo
bin/ed
bin/egrep
bin/false
bin/fatattr
bin/fdflush
bin/fgrep
bin/fsync
bin/getopt
bin/grep
bin/gunzip
bin/gzip
bin/hostname
bin/ionice
bin/iostat
bin/ipcalc
bin/kbd_mode
bin/kill
bin/ln
bin/login
bin/ls
bin/lzop
bin/makemime
bin/mkdir
bin/mknod
bin/mktemp
bin/more
bin/mount
bin/mountpoint
bin/mpstat
bin/mv
bin/netstat
bin/nice
bin/pidof
bin/ping
bin/ping6
bin/pipe_progress
bin/printenv
bin/ps
bin/pwd
bin/reformime
bin/rev
bin/rm
bin/rmdir
bin/run-parts
bin/sed
bin/setserial
bin/sh
bin/sleep
bin/stat
bin/stty
bin/su
bin/sync
bin/tar
bin/touch
bin/true
bin/umount
bin/uname
bin/usleep
bin/watch
bin/zcat
sbin/acpid
sbin/adjtimex
sbin/arp
sbin/blkid
sbin/blockdev
sbin/depmod
sbin/fbsplash
sbin/fdisk
sbin/findfs
sbin/fsck
sbin/fstrim
sbin/getty
sbin/halt
sbin/hdparm
sbin/hwclock
sbin/ifconfig
sbin/ifdown
sbin/ifenslave
sbin/ifup
sbin/init
sbin/inotifyd
sbin/insmod
sbin/ip
sbin/ipaddr
sbin/iplink
sbin/iproute
sbin/iprule
sbin/iptunnel
sbin/klogd
sbin/loadkmap
sbin/logread
sbin/losetup
sbin/lsmod
sbin/mdev
sbin/mkdosfs
sbin/mkfs.vfat
sbin/mkswap
sbin/modinfo
sbin/modprobe
sbin/nameif
sbin/nologin
sbin/poweroff
sbin/raidautorun
sbin/reboot
sbin/rmmod
sbin/route
sbin/setconsole
sbin/slattach
sbin/swapoff
sbin/swapon
sbin/switch_root
sbin/sysctl
sbin/syslogd
sbin/tunctl
sbin/udhcpc
sbin/vconfig
sbin/watchdog
usr/bin/[
usr/bin/[[
usr/bin/awk
usr/bin/basename
usr/bin/beep
usr/bin/blkdiscard
usr/bin/bunzip2
usr/bin/bzcat
usr/bin/bzip2
usr/bin/cal
usr/bin/chvt
usr/bin/cksum
usr/bin/clear
usr/bin/cmp
usr/bin/comm
usr/bin/crontab
usr/bin/cryptpw
usr/bin/cut
usr/bin/dc
usr/bin/deallocvt
usr/bin/diff
usr/bin/dirname
usr/bin/dos2unix
usr/bin/du
usr/bin/dumpleases
usr/bin/eject
usr/bin/env
usr/bin/expand
usr/bin/expr
usr/bin/find
usr/bin/flock
usr/bin/fold
usr/bin/free
usr/bin/ftpget
usr/bin/ftpput
usr/bin/fuser
usr/bin/groups
usr/bin/hd
usr/bin/head
usr/bin/hexdump
usr/bin/hostid
usr/bin/id
usr/bin/install
usr/bin/ipcrm
usr/bin/ipcs
usr/bin/killall
usr/bin/less
usr/bin/logger
usr/bin/lsof
usr/bin/lsusb
usr/bin/lzcat
usr/bin/lzma
usr/bin/lzopcat
usr/bin/md5sum
usr/bin/mesg
usr/bin/microcom
usr/bin/mkfifo
usr/bin/mkpasswd
usr/bin/nc
usr/bin/nmeter
usr/bin/nohup
usr/bin/nsenter
usr/bin/nslookup
usr/bin/od
usr/bin/openvt
usr/bin/passwd
usr/bin/patch
usr/bin/pgrep
usr/bin/pkill
usr/bin/pmap
usr/bin/printf
usr/bin/pscan
usr/bin/pstree
usr/bin/pwdx
usr/bin/readlink
usr/bin/realpath
usr/bin/renice
usr/bin/reset
usr/bin/resize
usr/bin/seq
usr/bin/setkeycodes
usr/bin/setsid
usr/bin/sha1sum
usr/bin/sha256sum
usr/bin/sha3sum
usr/bin/sha512sum
usr/bin/showkey
usr/bin/shuf
usr/bin/smemcap
usr/bin/sort
usr/bin/split
usr/bin/strings
usr/bin/sum
usr/bin/tac
usr/bin/tail
usr/bin/tee
usr/bin/telnet
usr/bin/test
usr/bin/tftp
usr/bin/time
usr/bin/timeout
usr/bin/top
usr/bin/tr
usr/bin/traceroute
usr/bin/traceroute6
usr/bin/truncate
usr/bin/tty
usr/bin/ttysize
usr/bin/udhcpc6
usr/bin/unexpand
usr/bin/uniq
usr/bin/unix2dos
usr/bin/unlink
usr/bin/unlzma
usr/bin/unlzop
usr/bin/unshare
usr/bin/unxz
usr/bin/unzip
usr/bin/uptime
usr/bin/uudecode
usr/bin/uuencode
usr/bin/vi
usr/bin/vlock
usr/bin/volname
usr/bin/wc
usr/bin/wget
usr/bin/which
usr/bin/whoami
usr/bin/whois
usr/bin/xargs
usr/bin/xzcat
usr/bin/yes
usr/sbin/addgroup
usr/sbin/add-shell
usr/sbin/adduser
usr/sbin/arping
usr/sbin/brctl
usr/sbin/chpasswd
usr/sbin/chroot
usr/sbin/crond
usr/sbin/delgroup
usr/sbin/deluser
usr/sbin/dnsd
usr/sbin/ether-wake
usr/sbin/fakeidentd
usr/sbin/fbset
usr/sbin/fdformat
usr/sbin/ftpd
usr/sbin/httpd
usr/sbin/inetd
usr/sbin/killall5
usr/sbin/loadfont
usr/sbin/lspci
usr/sbin/nanddump
usr/sbin/nandwrite
usr/sbin/nbd-client
usr/sbin/ntpd
usr/sbin/powertop
usr/sbin/rdate
usr/sbin/rdev
usr/sbin/readahead
usr/sbin/readprofile
usr/sbin/remove-shell
usr/sbin/rfkill
usr/sbin/sendmail
usr/sbin/setfont
usr/sbin/setlogcons
usr/sbin/udhcpd
```